bluetoothd: add page (#6158)
tldr-pages/tldr: pages/linux/bluetoothd.md

# bluetoothd

> Daemon to manage bluetooth devices.
> More information: <https://manned.org/bluetoothd>.

- Start the daemon:

`bluetoothd`

- Start the daemon, logging to stdout:

`bluetoothd --nodetach`

> - Start the daemon with a specific configuration file:

Reviewer: ```suggestion
- Start the daemon with a specific configuration file (defaults to `/etc/bluetooth/main.conf`):
```

> `bluetoothd --configfile {{path/to/file}}`

Reviewer: How about showing the default for a little variety and extra info?
```suggestion
`bluetoothd --configfile {{/etc/bluetooth/main.conf}}`
```

Reviewer: @navarroaxel

Author: Check this: https://github.com/tldr-pages/tldr/blob/main/pages/linux/openfortivpn.md#L10

Reviewer: I like that better.  I'll resolve this thread and start a new one with a suggestion.

> - Start daemon:
> 
> `bluetoothd`
> 
> - Start logging to stdout:
> 
> `bluetoothd --nodetach`
> 
> - Start with a specific configuration file:
> 
> `bluetoothd --configfile {{path/to/file}}`
> 
> - Start with verbose output to stderr:
> 
> `bluetoothd --debug`
> 
> - Start with verbose output for an specific file:

Reviewer: ```suggestion
- Start the daemon:

`bluetoothd`

- Start the daemon, logging to stdout:

`bluetoothd --nodetach`

- Start the daemon with a specific configuration file:

`bluetoothd --configfile {{path/to/file}}`

- Start the daemon with verbose output to stderr:

`bluetoothd --debug`

- Start the daemon with verbose output for an specific file:
```

> - Start the daemon with verbose output for an specific file:

Reviewer: ```suggestion
- Start the daemon with verbose output for a specific file:
```

> - Start the daemon with verbose output for a specific file:

Reviewer 2: What kind of file is this? A configuration file, or some other kind of file?

Author: source file, e.g. `profiles/a2dp.c`
The man page says:
> If file options are present, then only debug prints from that source file are printed. The option can be a pattern containing "*" and "?" characters.
>
> Example: --debug=src/adapter.c:src/agent.c

Reviewer: Oh weird.  Then maybe something like this?
```suggestion
- Start the daemon with verbose output coming from specific files in the bluetoothd source:
```

> `bluetoothd --debug={{path/to/file.c}}`

Reviewer: Why `.c`?
```suggestion
`bluetoothd --debug={{path/to/file}}`
```

> - Start the daemon with verbose output coming from specific files in the bluetoothd source:

Author: Oh! I'm not sure if this only applies to source code from `bluetoothd` because this support plugins too, so:
```suggestion
- Start the daemon with verbose output coming from specific files in the source:
```

Reviewer: Maybe
```suggestion
- Start the daemon with verbose output coming from specific files in the bluetoothd/plugins source:
```
?

Reviewer 2: ```suggestion
- Start the daemon with verbose output coming from specific files in the bluetoothd or plugins source:
```

> `bluetoothd --debug={{path/to/file}}`

Reviewer: And maybe this?
```suggestion
`bluetoothd --debug={{path/to/file1}}:{{path/to/file2}}:{{path/to/file3}}`
```

Author: How we are specifying this now? @marchersimon @CleanMachine1

Reviewer 2: This suggestion is fine @navarroaxel - as I mentioned in the issue we discussed this it's usually a case-by-case basis.
